### PR TITLE
Update installation instructions

### PR DESCRIPTION
**Pull request by ploh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ ETMoses is released under the [MIT License](LICENSE.txt).
 
 ## Installing
 
+* Install ImageMagick
+  * Ubuntu: `sudo apt-get install imagemagick libmagickwand-dev`
```
Comment (grdw) [inline README.md]: Could you add MacOSX way as well. That is:

`brew install imagemagick`

Comment (ploh) [inline README.md]: Done

**Comment by ploh** [comment]
@grdw Any more comments/suggestions about this?

**Comment by grdw** [comment]
Nope 👍

**Pull request merged**